IoT-Projects/SigfoxDevices/Sens'it v2.1/HTTP-SendData/SendData_Ubidots.js

```diff
@@ -318,7 +318,7 @@ function payloadDecode(data) {
 async function ubidotsRequest(token, label, payload) {
   const options = {
     method: 'POST',
-    url: `https://industrial.api.ubidots.com/api/v1.6/devices/${label}`,
+    url: 'https://industrial.api.ubidots.com/api/v1.6/devices/' + label,
```

Comment:
why do you need this change? It creates two string objects and I do not quite see the need to do this

Reply:
Thanks for the clarification. @d4vsanchez 

I honestly don't know why, as I didn't identify the issue personally. The change was requested by the CS team as I have permission to make changes in the repository.  They reported that after the NodeJS update the example stopped working, then after changing that line of code was working correctly. @jotathebest